utils/src/main/java/com/cloud/utils/script/Script.java

```diff
@@ -243,12 +243,19 @@ public String execute(OutputInterpreter interpreter) {
                         //process completed successfully
                         if (_process.exitValue() == 0) {
                             _logger.debug("Execution is successful.");
+                            String result;
```

Comment:
I guess these are for debugging, right ? @DaanHoogland

Reply:
yes, should I remove those? I think they could serve purpose for other work as well.